src/index.html

```diff
@@ -4,9 +4,35 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.8.2/css/bulma.min.css">
+  <link rel="stylesheet" href="style.css">
+  <title>social-network-recipes</title>
 </head>
 <body>
+<!-- TODO: Add SDKs for Firebase products that you want to use
+     https://firebase.google.com/docs/web/setup#available-libraries -->
+     <script src="https://www.gstatic.com/firebasejs/7.14.2/firebase.js"></script>
+     <script>
+       var firebaseConfig = {
+         apiKey: "AIzaSyBucI_PfC9w2g4RpcojM47YE9aEcJrdUrA",
+         authDomain: "social-network-recipes.firebaseapp.com",
+         databaseURL: "https://social-network-recipes.firebaseio.com",
+         projectId: "social-network-recipes",
+         storageBucket: "social-network-recipes.appspot.com",
+         messagingSenderId: "265483566973",
+         appId: "1:265483566973:web:4eacc08547367e47ed1614",
+         measurementId: "G-XTZT14HD22"
+       };
+       firebase.initializeApp(firebaseConfig);
+       firebase.analytics();
+     </script>
   <script type="module" src="index.js"></script>
+  <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
```

Comment:
Dos Frameworks de estilos causara conflictos, no es nada recomendable, decidan desde un inicio que utilizar ¿bulma o bootstrap? en caso de que cambien por uno o por otro se recomienda que desinstalen el otro(borrando todas las dependencias)
```suggestion
//bulma esta instalado arriba, en la linea 7 esta su enlace cdn
```